README.md

```diff
@@ -158,6 +158,89 @@ Set `googleAnalytics` in `config.toml` to activate Hugo's [internal Google Analy
 
 Set `disqusshortname` in `config.toml` to activate Hugo's [internal Disqus template](https://gohugo.io/templates/internal/#disqus).
 
+## Staticman
+
+Procedures to start using Staticman:
+
+1. Add Staticman bot/app, depending on your Git service provider.
+    - GitHub: choose either one of the following method
+        + GitHub App: refer to issue
+        [https://github.com/eduardoboucas/staticman/issues/243](https://github.com/eduardoboucas/staticman/issues/243)
+        for detailed procedures.
+        + GitHub bot: invite
+        **[@staticmanlab](https://github.com/staticmanlab)** as a collaborator
+        to your repository (by going to your repository's **Settings** page,
+        navigating to the **Collaborators** tab.
+
+            ![add staticmanlab on GitHub](https://user-images.githubusercontent.com/5748535/62385579-c5fc3a80-b555-11e9-9a87-a71e00022d13.png)
+
+            Then help **[@staticmanlab](https://github.com/staticmanlab)**
+            accept the invitation by going to
+
+                https://staticman3.herokuapp.com/v3/connect/github/<username>/<repo-name>
```

Comment:
[Staticman's documentation](https://staticman.net) only covers v2, so there's a need to give a self-contained setup guide.